server/routes/notification.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const {
+  createNotification,
+  markNotificationRead,
+  getAllNotifications,
+} = require("../controllers/notification");
+
+router.route("/create").post(protect, createNotification);
```

Comment:
No need to use `CRUD` names in the routes, these should be inferred from the methods instead. As stated by [this](https://restfulapi.net/resource-naming/) guide:
> We should not use URIs to indicate a CRUD function. URIs should only be used to uniquely identify the resources and not any action upon them.
>
> We should use HTTP request methods to indicate which CRUD function is performed.

```suggestion
router.route("/").post(protect, createNotification);
```